youtube_dl/extractor/neteasemusic.py

```diff
@@ -108,30 +106,55 @@ class NetEaseMusicIE(NetEaseMusicBaseIE):
             'ext': 'mp3',
             'title': '소원을 말해봐 (Genie)',
             'creator': '少女时代',
-            'description': 'md5:79d99cc560e4ca97e0c4d86800ee4184',
             'upload_date': '20100127',
             'timestamp': 1264608000,
             'alt_title': '说出愿望吧(Genie)',
         }
     }]
 
+    def _fix_timestamp(self, timestamp):
+        # Netease returns timestamp use 2 or 3 digits for less than
+        # 1 second metrics
+        # While standard LRC requires exact 2 digits
+        # remvoe the 3rd digit if the there're three
+        match_expr = r'\[([0-9]{2}:[0-9]{2}\.[0-9]{2})(.*)\]'
+        match_result = re.match(match_expr, timestamp)
+        if match_result:
+            # This must match as match_expr is exact the first group of lyrics_expr
+            return "[" + match_result.group(1) + "]"
+        else:
+            # return a valid timestamp to avoid other applications' error
+            return "[00:00.00]"
+
+    def _fix_lyric_timestamp(self, line):
+        text = line
+        corrected_times = ""
+        lyrics_expr = r'(\[[0-9]{2}:[0-9]{2}\.[0-9]{2,}\])([^\n]*)'
+        # Handle time tag with one or more time in one line.
```

Comment:
URL for multiple time in one line is `http://music.163.com/#/song?id=4926366`
added as a test in latest commit